app/middleware/setLocals.js

```diff
@@ -2,9 +2,9 @@ const backLinkUtils = require('../lib/backLinkUtils');
 
 function fromRequest(req, res, next) {
   /* eslint-disable no-param-reassign */
-  res.locals.location = req.query.location;
+  res.locals.search = req.query.search;
   res.locals.context = req.query.context || '';
```

Comment:
At the content review yesterday it was discussed that there is a need to show from where the user is coming from, so the context is still used to display the back to a specific condition/symptom a la pharmacy finder